Cargo.toml

```diff
@@ -11,21 +11,33 @@ keywords = ["renderer", "first person shooter", "3D", "rust", "wgpu", "game"]
 categories = ["rendering", "game framework", "gamedev"]
 publish = false
 
+[features]
+default = []
+#Doesn't work anymore, client version not aligned? #profile-with-puffin = ["profiling/profile-with-puffin", "puffin_http"]
```

Comment:
I think puffin_viewer had a new release recently, maybe they broke something or we need to update our puffin version or you need to upgrade to the latest puffin_viewer version

Reply:
Yes, I tried a version that I thought was correct without success. For now I would say to ignore the problem: puffin does not give us anything more than tracy (we could also avoid supporting it)

Reply:
I've removed support for puffin, I see no point in keeping it. I'm curious to try the others sometime. I noticed that tracing is supported by default in Bevy instead. It makes sense, since it's developed by tokio developers, its functionality goes along with Bevy's design.